### PR TITLE
Correlation by Column doc

### PR DESCRIPTION
**Pull request by hidekoji**
Added Correlation by Column doc.

### CODE REVIEW

File: analytics/correlation_category.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Correlation by Category
+
+Calculates Correlation by Category.
```
Comment (hideaki) [inline analytics/correlation_category.md]: I think we can be a little more descriptive, for example,
`Calculates correlations among groups defined by a category column.`

File: analytics/correlation_category.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Input Data
+Input data should contain categorical columns and numeric columns.
+
+  * Category Column - Categorical column that you want to see correlation.
```
Comment (hideaki) [inline analytics/correlation_category.md]: Categorical column that defines the groups among which correlations are calculated.

File: analytics/correlation_category.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  * Measured By Column - key column to aggregate measure.
```
Comment (hideaki) [inline analytics/correlation_category.md]: Column that defines dimension to calculate correlation. For each group, measure is aggregated for each value of this column, forming a vector that represents the group. Then, correlations are calculated among these vectors. If Date/POSIXct column is specified, measure is aggregated for each period of time (day, week, etc.) that is specified.

File: analytics/correlation_category.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+2. If necessary, click "+" button on the left of existing Analytics tabs, to create a new Analytics.
+3. Select "Correlation by Categories" for Analytics Type.
+4. Select Category Columns that you want to see correlation.
+5. Select Measured By Columns that you want to summarize measure column.
+6. Select Measure Column that you want to use to calculate correlation.
+7. Click Run button to run the analytics.
+8. Select view type (explained below) by clicking view type link to see each type of generated visualization.
+
+### "Correlation Matrix" View
+"Correlation Matrix" View displays correlation with Heatmap. Red color means it has positive correlation and Blue color means it has negative correlation. The darker the color, the stronger the correlation.
```
Comment (hideaki) [inline analytics/correlation_category.md]: "correlation" in the following sentence should be in plural.
"Correlation Matrix" View displays correlation with Heatmap.

File: analytics/correlation_category.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+![](images/cor_by_category_matrix.png)
+
+### "Positive Correlations" View
+"Positive Correlations" View displays the positive correlation value for column pair.You can click Column Header for Correlation to sort the data by Correlation value.
```
Comment (hideaki) [inline analytics/correlation_category.md]: This is not for column pair.

File: analytics/correlation_category.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+![](images/cor_by_category_positive.png)
+
+### "Negative Correlations" View
+"Negative Correlations" View displays the negative correlation value for column pair.You can click Column Header for Correlation to sort the data by Correlation value.
```
Comment (hideaki) [inline analytics/correlation_category.md]: This is not for column pair.

File: analytics/correlation_category.md (hunk 1 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed as suggested.

File: analytics/correlation_category.md (hunk 2 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed as suggested.

File: analytics/correlation_category.md (hunk 3 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed as suggested.

File: analytics/correlation_category.md (hunk 4 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed as suggested.

File: analytics/correlation_category.md (hunk 5 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed to value pair.

File: analytics/correlation_category.md (hunk 6 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed to value pair.

File: analytics/correlation_column.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Correlation by Column
+
+Calculates Correlation by Column.
```
Comment (hideaki) [inline analytics/correlation_column.md]: Calculates Correlations among columns.

File: analytics/correlation_column.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## Input Data
+Input data should contain numeric columns.
+
+  * Variable Columns - Numeric columns in which you want to see correlation.
```
Comment (hideaki) [inline analytics/correlation_column.md]: Numeric columns among which correlations are calculated.

File: analytics/correlation_column.md (hunk 7 above)
Comment (hidekoji) [reply analytics/correlation_column.md]: Changed as suggested.

File: analytics/correlation_column.md (hunk 8 above)
Comment (hidekoji) [reply analytics/correlation_column.md]: Changed as suggested

File: analytics/correlation_category.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+7. Click Run button to run the analytics.
+8. Select view type (explained below) by clicking view type link to see each type of generated visualization.
+
+### "Correlation Matrix" View
+"Correlation Matrix" View displays correlations with Heatmap. Red color means it has positive correlation and Blue color means it has negative correlation. The darker the color, the stronger the correlation.
+
+![](images/cor_by_category_matrix.png)
+
+### "Positive Correlations" View
+"Positive Correlations" View displays the positive correlation value for each value pair. You can click Column Header for Correlation to sort the data by Correlation value.
```
Comment (hideaki) [inline analytics/correlation_category.md]: "Positive Correlations" View displays the most positive 100 correlations.

File: analytics/correlation_category.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+![](images/cor_by_category_positive.png)
+
+### "Negative Correlations" View
+"Negative Correlations" View displays the negative correlation value for each value pair. You can click Column Header for Correlation to sort the data by Correlation value.
```
Comment (hideaki) [inline analytics/correlation_category.md]: "Negative Correlations" View displays the most negative 100 correlations.

File: analytics/correlation_column.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+6. Click Run button to run the analytics.
+7. Select view type (explained below) by clicking view type link to see each type of generated visualization.
+
+### "Correlation Matrix" View
+"Correlation Matrix" View displays correlations with Heatmap. Red color means it has positive correlation and Blue color means it has negative correlation. The darker the color, the stronger the correlation.
+
+![](images/cor_by_column_matrix.png)
+
+### "Positive Correlations" View
+"Positive Correlations" View displays the positive correlation value for column pair. You can click Column Header for Correlation to sort the data by Correlation value.
```
Comment (hideaki) [inline analytics/correlation_column.md]: Positive Correlations" View displays the most positive 100 correlations.

File: analytics/correlation_column.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+![](images/cor_by_column_positive.png)
+
+### "Negative Correlations" View
+"Negative Correlations" View displays the negative correlation value for column pair. You can click Column Header for Correlation to sort the data by Correlation value.
```
Comment (hideaki) [inline analytics/correlation_column.md]: Negative Correlations" View displays the most negative 100 correlations.

File: analytics/correlation_column.md (hunk 11 above)
Comment (hidekoji) [reply analytics/correlation_column.md]: Changed as suggested.

File: analytics/correlation_column.md (hunk 12 above)
Comment (hidekoji) [reply analytics/correlation_column.md]: Changed as suggested.

File: analytics/correlation_category.md (hunk 10 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed as suggested.

File: analytics/correlation_category.md (hunk 9 above)
Comment (hidekoji) [reply analytics/correlation_category.md]: Changed as suggested.

File: analytics/var_importance.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+![](images/var_importance.png)
+
+### "Importance Table" View
+"Importance Table" View displays the importance in table format with color as indicator of importance. By clicking column header for Importance, you can sort data.
+
+![](images/var_importance_table.png)
+
+### "Model Quality" View
+"Model Quality" View displays the quality of the model created for this Variable Importance Analytics. Each row shows the model performance from the point whether the Class (i.e. Value in What to Predict column) prediction was correct or not if What to Predict is categorical column.
```
Comment (hideaki) [inline analytics/var_importance.md]: There should be explanation for the case when the target column is numeric.

File: analytics/var_importance.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+
+- F Score - A measure of Test Accuracy. The score ranges between 0 and 1 and Higher is better.  It's harmonic mean of precision and recall.
+- Accuracy Rate - Another measure of Test Accuracy, which is calculated as `(Total True Positive + total True Negative) / Total Popluation.`
+- Misclassification Rate - A rate the model fails to classify the correct value. (i.e. 1- Accuracy rate)
```
Comment (hideaki) [inline analytics/var_importance.md]: `The rate the model fails to classify correctly. (i.e. 1 - Accuracy Rate)`

File: analytics/var_importance_time.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+7. Click Run button to run the analytics.
+8. Select view type (explained below) by clicking view type link to see each type of generated visualization.
+
+### "Impact" View
+"Importance" View displays parameter estimate and confidence interval information on scatter plot. Blue means "Less Likely to Occur" and Orange means "More Likely to Occur". 
+
+![](images/var_importance_with_time_impact.png)
+
+### "High Confidence" View
+"High Confidence" View displays the variables whose both confidence interval edges are more than 1 (or less than 1). With this thresh hold, we can see only variables that have clear impact on the event.
```
Comment (hideaki) [inline analytics/var_importance_time.md]: thresh hold -> threshold

`... we can see only variables that have clear direction of impact on the event.`

File: analytics/var_importance.md (hunk 13 above)
Comment (hidekoji) [reply analytics/var_importance.md]: Added explanation for a case when What to Predict column is numeric column.

File: analytics/var_importance.md (hunk 14 above)
Comment (hidekoji) [reply analytics/var_importance.md]: Changed as suggested.

File: analytics/var_importance_time.md (hunk 15 above)
Comment (hidekoji) [reply analytics/var_importance_time.md]: Changed as suggested.
